source/server/getting-started/implementations/spongeforge.rst

```diff
@@ -69,10 +69,17 @@ Installing SpongeForge
 
     If you use (or are planning to use) a game server host, they may have a control panel that can install Sponge for
     you.
+.. note::
+
+  When using the Mojang installer, Mojang makes use of their own Java version and not the one you installed on your
+  system. The installer currently ships with Java ``1.8.0_25`` for Windows and ``1.8.0_60`` for macOS. Note that Sponge
+  requires **at least** ``1.8.0_20`` or above to run properly.
```

Comment:
Maybe add a comment such as `but it is recommended to use the latest Java 8 version`